pages/volunteer/Volunteer.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.controller('volunteerPage', [
+  '$scope', function($scope) {
+    var designSlideQuotes = [
```

Comment:
Make this more declarative. Have a JS global constant in its own file that's something like:

```
oppiaFoundationWebsite.constant('VOLUNTEER_PAGE_SECTIONS', [{
  category: 'design',
  description: ...,
  volunteers: [{
    name: ...,
    quote: ...,
    profilePictureImgFilename: ...,
    ...
  }]
}, {
  ...
}]);
```

Then only this file needs to be modified if we wanted to update the page later to keep it current.

Bonus points for adding Karma tests that validate the format of the constant. This can also be done later, though.

Reply:
An alternative, if you want to keep the non-volunteer-details text hardcoded, is to do something like:

```
oppiaFoundationWebsite.constant('VOLUNTEER_INFO', {
  design: [{
    name: ...,
    quote: ...,
    profilePictureImgFilename: ...,
    ...
  }],
  ...
});
```

and retrieve the volunteer info by category. That might be better actually since it's this part (and not the overall descriptions) that are more likely to be edited.

Reply:
Are we updating each of the __tab__ (development, design, ...) every so often? 

I think only the personal info slides should be constantly maintained. The page itself should be edited via the HTML template.

If we are only updating the Volunteer page's __personal info__ and not the whole "what we do" or what "development" is about, I agree with the 2nd suggestion: creating a constant class for the `VOLUNTEER_INFO`

I am OK with the first suggestion too. However, parsing for paragraphing, columns and custom styling is going to prove challenging if we do something like
```javascript
oppiaFoundationWebsite.constant('VOLUNTEER_PAGE_TABS', [{
  tab: 'design',
  description: ...,
  what_we_do: ...,
  volunteers: [{
    name: ...,
    quote: ...,
    profilePictureImgFilename: ...,
    ...
  }]
}],
...
```
Changing the HTML template for the description or the "what we do" portion should be done via HTML template imo.
Is there anything I am missing before I plunge into this endeavor?

Reply:
Right -- for similar reasons I agree with the 2nd suggestion more.

Reply:
Done!